cmd/gokr-packer/write.go

```diff
@@ -447,3 +495,16 @@ func writeMBR(f io.ReadSeeker, fw io.WriteSeeker, partuuid uint32) error {
 
 	return nil
 }
+
+// getDuplication between the two given filesystems
+func getDuplication(fiA, fiB *fileInfo) (paths []string) {
+	allPaths := append(fiA.pathList(), fiB.pathList()...)
+	checkMap := make(map[string]struct{}, len(allPaths))
```

Comment:
a map[string]bool would make for slightly nicer to read code here

Reply:
with struct the memory footprint should be little lower but I think in this case it is not so important